@here/harp-mapview/lib/MapView.ts

```diff
@@ -1897,33 +1897,23 @@ export class MapView extends THREE.EventDispatcher {
 
         const renderList = this.m_visibleTiles.dataSourceTileList;
 
-        renderList.forEach(({ zoomLevel, renderedTiles, visibleTiles, numTilesLoading }) => {
-            // The visible tiles may not actually be rendered, but they have to stay in the cache
-            // for the next frame.
-            visibleTiles.forEach(tile => {
-                tile.frameNumLastVisible = this.m_frameNumber;
```

Comment:
This was used to keep the tiles in the cache, but now we use when the given tile was requested as the means of deciding when to evict from the cache.

Reply:
So this commit also contains function changes. Please rename the commit message or ideally split into two commits (renaming and functional)